pkg/dockerregistry/server/metrics/metrics.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	registryNamespace = "openshift"
+	registrySubsystem = "registry"
+)
+
+var (
+	RegistryAPIRequests *prometheus.SummaryVec
+)
+
+// Register the metrics.
+func Register() {
+	RegistryAPIRequests = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Namespace: registryNamespace,
+			Subsystem: registrySubsystem,
+			Name:      "request_duration_microseconds",
+			Help:      "Request latency summary in microseconds for each operation",
```

Comment:
@smarterclayton Changed it to seconds. But `http_request_duration_microseconds` will remain in place.  We have really old version of `github.com/prometheus/client_golang/prometheus` in the vendor and `InstrumentHandler` uses microseconds as unit, which is deprecated in new version and should be replaced by seconds.